src/core/polyhedral/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+IF(DEFINED CILK_PATH AND NOT ${CILK_PATH} STREQUAL "")
```

Comment:
can you make this part of `src/Cmakelists.txt` ? 
polyhedral files are also built there and it's confusing to add this new file now